packages/ui/src/routes/hardware-wallet/RemoveDevice.tsx

```diff
@@ -71,48 +74,73 @@ const HardwareWalletRemoveDevicePage = () => {
                         onClick={() =>
-                            isDeviceImported(Devices.LEDGER) &&
-                            setSelectedVendor(Devices.LEDGER)
+                            importedLedger && setSelectedVendor(Devices.LEDGER)
                         }
                         className={classnames(
                             "bg-white rounded-md p-4 w-1/2 flex flex-col items-center justify-center space-y-3 cursor-pointer border hover:border-primary-300",
                             selectedVendor === Devices.LEDGER
                                 ? "border-primary-300"
                                 : "border-primary-100",
-                            !isDeviceImported(Devices.LEDGER) &&
-                                "opacity-50 pointer-events-none"
+                            !importedLedger && "opacity-50 pointer-events-none"
                         )}
                         style={{ height: "120px" }}
                     >
                         <img
                             src={ledger}
-                            alt="Connect Ledger"
-                            className="h-8"
+                            alt="Remove Ledger hardware wallet"
+                            className="h-8 mb-2"
                         />
-                        {isDeviceImported(Devices.LEDGER)} Accounts
+                        {importedLedger === 1
+                            ? "1 Account"
+                            : importedLedger + " accounts"}
                     </button>
                     <button
                         type="button"
                         onClick={() =>
-                            isDeviceImported(Devices.TREZOR) &&
-                            setSelectedVendor(Devices.TREZOR)
+                            importedTrezor && setSelectedVendor(Devices.TREZOR)
                         }
                         className={classnames(
                             "bg-white rounded-md justify-center p-4 w-1/2 flex flex-col items-center group space-y-3 cursor-pointer border hover:border-primary-300",
                             selectedVendor === Devices.TREZOR
                                 ? "border-primary-300"
                                 : "border-primary-100",
-                            !isDeviceImported(Devices.TREZOR) &&
-                                "opacity-50 pointer-events-none"
+                            !importedTrezor && "opacity-50 pointer-events-none"
                         )}
                         style={{ height: "120px" }}
                     >
                         <img
                             src={trezor}
-                            alt="Connect Trezor"
-                            className="h-8"
+                            alt="Remove Trezor hardware wallet"
+                            className="h-8 mb-2"
+                        />
+                        {importedTrezor === 1
+                            ? "1 Account"
```

Comment:
` "1 Account" vs "N accounts" ` => please be consistent with the capitalization. I would capitalize the A to match the rest of the extension